dlk/python/dlk/templates/include/de10_nano.h

```diff
@@ -146,9 +146,14 @@ Parameters calcParameters(uint32_t inputHeight, uint32_t inputWidth, uint32_t in
   constexpr uint32_t maxBurst = 32;
   constexpr uint32_t b = 32;
 
-  assert((kernelHeight == 3 && kernelWidth == 3) || (kernelHeight == 1 && kernelWidth == 1));
-
-  uint32_t pad = (kernelHeight == 1) ? 0 : 1;
+  constexpr std::size_t n_bit = 2;
+  constexpr std::size_t maxA = (1 << n_bit) - 1;
+  assert(kernelHeight == kernelWidth); // kernel rectangle must be square
+  assert(kernelHeight % 2 == 1); // kernel size must be even
```

Comment:
the comment should be 'must be odd' I think?

Reply:
Oh! yes!

Reply:
ahh, I should not be reviewer, I just happened to see by chance.

Reply:
Fixed.